tools/nx-plugins/npm-package/package.json

```diff
@@ -1,5 +1,6 @@
 {
 	"name": "@csnx/npm-package",
+	"version": null,
```

Comment:
Is this intentional? If so I do not think you want a CHANGELOG at all…

Reply:
also removed